docs/source/en/using-diffusers/weighted_prompts.mdx

```diff
@@ -0,0 +1,101 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Weighting prompts
+
+Text-guided diffusion models generate images based on a given text prompt. Often, the text prompt
+can include multiple concepts that the model should generate and it's often desireable to weight
+certain parts of the prompt more or less. 
+
+Diffusion models function by conditioning the cross attention layers of the diffusion model with contextualized text embeddings via the cross attention layers (see the [Stable Diffusion Guide for more information](./stable-diffusion)).
+Thus a simple way to emphasize (or de-emphasize) certain parts of the prompt is by increasing or reducing the scale of the text embedding vector that correspond to the part of the prompt.
+This is called "prompt-weighting" and has been a highly demanded feature by the community (see issue [here](https://github.com/huggingface/diffusers/issues/2431)).
+
+## How to do prompt-weighting in Diffusers
+
+Generally speaking, we do not consider prompt weighting as a core feature of diffusers since the philosophy of diffusers 
+is to be a toolbox that enables other libraries, such as [InvokeAI](https://github.com/invoke-ai/InvokeAI) or [diffuzers](https://github.com/abhishekkrthakur/diffuzers) to build powerful UIs. Therefore, `diffusers` 
+simply exposes a [`prompt_embeds`](https://huggingface.co/docs/diffusers/v0.14.0/en/api/pipelines/stable_diffusion/text2img#diffusers.StableDiffusionPipeline.__call__.prompt_embeds) function argument to many pipelines such as [`StableDiffusionPipeline`] allowing to directly pass the "prompt-weighted"/scaled text embeddings to the pipeline.
+
+As it's not super simple to process text embeddings to example or de-emphasize certain parts of the prompt,
+we strongly recommend to rely on the [compel library](https://github.com/damian0815/compel) to handle the prompt-weighting for you.
```

Comment:
```suggestion
The [compel library](https://github.com/damian0815/compel) provides an easy way to emphasize or de-emphasize portions of the prompt for you. We strongly recommend it instead of preparing the embeddings yourself.
```